massa-event-cache/src/controller.rs

```diff
@@ -0,0 +1,43 @@
+use crate::config::EventCacheConfig;
+use crate::event_cache::EventCache;
+use massa_models::execution::EventFilter;
+use massa_models::output_event::SCOutputEvent;
+
+#[allow(dead_code)]
```

Comment:
useful?

Reply:
I've moved it to the cfg field only otherwise the compiler complains it is never read